src/utils/keyRotation.ts

```diff
@@ -0,0 +1,58 @@
+import { Bindings } from '../bindings';
+import cronParser from 'cron-parser';
+
+export const WILDCARD = '*';
+
+export function shouldRotateKey(date: Date, env: Bindings): boolean {
+	const utcDate = new Date(date.toISOString());
+
+	if (env.ROTATION_CRON_STRING) {
+		const isRotationCronTime = matchCronTime(env.ROTATION_CRON_STRING, utcDate);
```

Comment:
what's the precision for the match? Ideally, we'd like it to work well with the cron of the worker (which is different from the cron here). Worker cron triggers the schedule event, which rotate if it matches `ROTATION_CRON_STRING` and clear otherwise